htdocs/comm/propal/card.php

```diff
@@ -193,8 +193,27 @@
 					}
 				}
 
-				$result = $object->createFromClone($user, $socid, (GETPOSTISSET('entity') ? GETPOST('entity', 'int') : null));
+				$result = $object->createFromClone($user, $socid);
 				if ($result > 0) {
+					$soc = new  Societe($db);
+					$obj = $soc->fetch(GETPOST('socid'));
+					if ($obj == 1 && $soc->tva_assuj != 1) {
```

Comment:
There is a confusion on what mean this parameter.
It does not means that you must use vat 0 when selling to this customer but that this customer use a VAT 0 when he sell products (for example a micro-enterprise sell product with vat 0, so its field tva_assuj is set but when you make an invoice, you create invoice with vat 0). There is for the moment no information to say "When you make invoice to this customer we must sell with VAT 0 to this customer"). 

The rule to know the vat to use for a customer is defined into get_default_tva()

So you can call get_default_tva() to know what is the VAT we must use for this customer, and if 0 instead of the one used after clone, then you can force to 0.